build-system/tasks/e2e/describes-e2e.js

```diff
@@ -343,18 +405,19 @@ class EndToEndFixture {
 /**
  * Get the controller object for the configured engine.
  * @param {!DescribesConfigDef} describesConfig
+ * @param {string} browserName
  */
 async function getController({
   engine = 'selenium',
   headless = false,
-}) {
+}, browserName) {
   if (engine == 'puppeteer') {
     const browser = await createPuppeteer({headless});
     return new PuppeteerController(browser);
   }
 
   if (engine == 'selenium') {
-    const driver = await createSelenium({headless});
+    const driver = await createSelenium(browserName, {headless});
```

Comment:
Should `browserName` be part of the `DescribesConfigDef`? I'm not seeing how the test author would specify that a test should run only in one browser or in all

Reply:
good question, would we want that to override individual e2e configurations if passed in as a flag?

Reply:
Hmm not sure, override or run the ones that opted into that browser. Trying to imagine what the use cases would be... could be helpful if we want to split up the Travis job by browser.

Reply:
usually tests will opt-out of a browser if they know they don't support them, so I don't know that we want a command line flag to override them necessarily. I think for a command line flag we'd want it to only run the tests that have opted-in to the browser(s) specified by the command line flags

Reply:
@cvializ thats a good point. ill follow up with the feature on a separate PR

added issue here https://github.com/ampproject/amphtml/issues/22432